### ISSUE TITLE
PCC drop in Openpose V2 Model

### ISSUE THREAD

**Issue by kamalrajkannan78** [tenstorrent/tt-xla#2417]
### Describe the bug

- `openpose/v2/pytorch-single_device-full-inference ` fails with `PCC=0.9080588817596436`. [dec3_openpose_v2.log](https://github.com/user-attachments/files/23901147/dec3_openpose_org.log)

**Comment by kamalrajkannan78** [comment]
FYI @sdjordjevicTT , This PR  https://github.com/tenstorrent/tt-xla/pull/2528 fixed the double run issue.

**Comment by kamalrajkannan78** [comment]
**15/12/2025**

### PCCs on current main

- Whole model - `0.9080588817596436`
- case 1 : conv2d + batchnorm2d - `0.9880387783050537 `
- case 2 : conv2d - `0.9997682571411133`
- case 3 : batchnorm2d -  `0.999877393245697`
- This indicates the failure occurs only when these two ops are combined.

### Logs:

- [dec15_openpose_c1.log](https://github.com/user-attachments/files/24161967/dec15_openpose_c1.log)
- [dec15_openpose_c2.log](https://github.com/user-attachments/files/24161969/dec15_openpose_c2.log)
- [dec15_openpose_c3.log](https://github.com/user-attachments/files/24161968/dec15_openpose_c3.log)
- [dec15_openpose.log](https://github.com/user-attachments/files/24161970/dec15_openpose.log)
- [conv2d_batchnorm2d_dec15_emitpy.zip](https://github.com/user-attachments/files/24162133/conv2d_batchnorm2d_dec15.zip)

**Comment by kamalrajkannan78** [comment]
## 20/1/2026

### PCCs on current main

- Whole model - `0.9180459677147291`
- case 1 : conv2d + batchnorm2d - `0.9994961321298823`
- case 2 : conv2d - `1.0`
- case 3 : batchnorm2d - `0.999993788747307`
- problematic sanity passing with valid PCC. Hence issue is from some other ops.

### Logs

- [jan20_openposev2_pcc_drop.log](https://github.com/user-attachments/files/24728382/jan20_openposev2_pcc_drop.log)
- [jan20_openposev2_c1.log](https://github.com/user-attachments/files/24728381/jan20_openposev2_c1.log)
- [jan20_openposev2_c2.log](https://github.com/user-attachments/files/24728380/jan20_openposev2_c2.log)
- [jan20_openposev2_c3.log](https://github.com/user-attachments/files/24728383/jan20_openposev2_c3.log)

### Repro

```
git checkout kkannan/jan20_openpose
git submodule update --init --recursive
git lfs pull --include '*.pt'

# Model run
pytest tests/runner/test_models.py::test_all_models_torch["openpose/v2/pytorch-single_device-inference"] -svv

# Sanity run 
# Place the contents from the respective scripts (case1, case2, case3 scripts present in description) into their loaders and rerun above command
```
